app/enterprise/1.3-x/developer-portal/structure-and-file-types.md

```diff
@@ -182,20 +202,35 @@ Each theme file is compromised of a few different folders:
 - **theme.conf.yaml**
   - This config file sets color and font defaults available to templates for reference as css variables.  It also determines what options are available in the Kong Manager Appearance page.
 
-#### assets
-The asset folder contains css/js/fonts/images for your templates to reference.  To access asset files from your templates, keep in mind that Kong assumes a path from the root of your selected theme.
+#### Theme Assets
+
+##### Path
+- **format:** `theme/*/assets/**/*`
+
+##### Description
+The asset folder contains css/js/fonts/images for your templates to reference.
+
```

Comment:
Let's add that base64 encoded strings at `theme/*/assets/images` will be base64 decoded on access.